src/executor/select_executor_test.go

```diff
@@ -411,7 +411,7 @@ func TestJoinExecutor(t *testing.T) {
 	results := []string{
 		"[[3 go] [5 lang]]",
 		"[[3 go]]",
-		"[[3 go 3] [4  ] [5  ] [6 lang 5]]",
```

Comment:
the want result should be [[3 go 3] [4  ] [5  ] [6 lang 5]]
and the result on MySQL is [[3 go 3] [4  ] [5  ] [6 lang 5]]